contrib/mesos/pkg/executor/executor.go

```diff
@@ -415,9 +415,18 @@ func (k *Executor) handleChangedApiserverPod(pod *api.Pod) {
 	}
 
 	oldPod := k.pods[task.podName]
+	if oldPod == nil {
+		// TODO(jdef) should sent a TASK_FAILED or TASK_LOST here
+		return
+	}
+
```

Comment:
should check if `oldPod.podName == ""` because that indicates we haven't sent this pod up to the kubelet yet (it's still processing in `launchTask()`